src/test/create-user.test.ts

```diff
@@ -13,13 +13,17 @@ describe('Create User Mutation', () => {
 
     const responseOutput = response.data.data.createUser;
 
-    const userFromDB = await userRepository.findOneBy({ id: responseOutput.id });
+    const userFromDb = await userRepository.findOneBy({ id: responseOutput.id });
 
-    expect(DEFAULT_USER_INPUT.name).to.be.deep.eq(userFromDB.name);
-    expect(DEFAULT_USER_INPUT.email).to.be.deep.eq(userFromDB.email);
-    expect(passwordHashing(DEFAULT_USER_INPUT.password)).to.be.deep.eq(userFromDB.password);
-    expect(DEFAULT_USER_INPUT.birthDate).to.be.deep.eq(userFromDB.birthDate);
-    expect(responseOutput).to.be.deep.eq(getExpectedUserOutput(userFromDB));
+    const { password, ...expectedResponse } = userFromDb;
+
+    expect(responseOutput).to.be.deep.eq(expectedResponse);
+
+    expect(DEFAULT_USER_INPUT.name).to.be.deep.eq(userFromDb.name);
+    expect(DEFAULT_USER_INPUT.email).to.be.deep.eq(userFromDb.email);
+    expect(passwordHashing(DEFAULT_USER_INPUT.password)).to.be.deep.eq(userFromDb.password);
+    expect(DEFAULT_USER_INPUT.birthDate).to.be.deep.eq(userFromDb.birthDate);
```

Comment:
n daria pra transformar em um objeto? algo assim:
```suggestion
    expect({...DEFAULT_USER_INPUT, password: passwordHashing(DEFAULT_USER_INPUT.password)}).to.be.deep.eq(userFromDb);
```

Reply:
Daria sim, mas ou seria preciso tirar o id de userFromDb:

`const { id, ...userFromDbWithoutId } = userFromDb;

    expect({ ...DEFAULT_USER_INPUT, password: passwordHashing(DEFAULT_USER_INPUT.password) }).to.be.deep.eq(
      userFromDbWithoutId,
    );`

Ou colocar um Id no DEFAULT_USER_INPUT
`expect({
      ...DEFAULT_USER_INPUT,
      password: passwordHashing(DEFAULT_USER_INPUT.password),
      id: userFromDb.id,
    }).to.be.deep.eq(userFromDb);`